doap.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<rdf:RDF
+        xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+        xmlns:foaf="http://xmlns.com/foaf/0.1/"
+        xmlns:xmpp="https://linkmauve.fr/ns/xmpp-doap#"
+        xmlns:schema="https://schema.org/"
+        xmlns="http://usefulinc.com/ns/doap#">
+    <Project rdf:about="https://xmpppy.sourceforge.net/">
+        <name>xmpppy</name>
+        <shortdesc xml:lang="en">Python 2/3 implementation of XMPP (RFC3920, RFC3921).</shortdesc>
+        <homepage rdf:resource="https://xmpppy.sourceforge.net/"/>
+        <programming-language>Python</programming-language>
+        <license rdf:resource="https://raw.githubusercontent.com/xmpppy/xmpppy/refs/heads/master/LICENSE"/>
```

Comment:
The DOAP schema says "The URI of an RDF description of the license the software is distributed under. E.g. a SPDX reference", so I don't think this would pass validation, but it might be good enough? I don't know of an RDF description for GPL-3. The DOAP repo doesn't have version specific files for licenses (I'm not even sure how to reference the non-versioned files).

Reply:
I think it's better to have a direct link to the LICENSE.
The validator can easily detect the license type if needed.
In the same time the LICENSE is the legal document, I guess just saying a license type may be not enough.

This lib uses the general http://usefulinc.com/doap/licenses/gplv3
https://xmpp.org/doap/python-nbxmpp.doap
Another use a link to the license in sources:
https://xmpp.org/doap/slixmpp.doap

The licenses aren't shown on the software pages:
https://xmpp.org/software/?category=libraries

It would be better to discuss in the jdev conference. But for now this is not a blocking issue.